kubernetes/EKS/.helmignore

```diff
@@ -0,0 +1,5 @@
+../templates/eks_cluster.yaml
```

Comment:
Now that we have a common directory in Helm to share some files, let's remove these files that are no longer needed:
- EKS/.helmignore (not needed since our values.yaml now lives in /Helm)
- EKS/Chart.yaml (same as above)
- EKS/templates/torchserve.yaml (same as above)

Reply:
@jeremiahschung Is this file, EKS/values.yaml,  still needed?

Reply:
Good catch! EKS/values.yaml is replaced by Helm/values.yaml so we can remove it as well.